render/egl.c

```diff
@@ -172,6 +172,15 @@ struct wlr_egl *wlr_egl_create(EGLenum platform, void *remote_display) {
 		return NULL;
 	}
 
+#ifdef EGL_KHR_platform_gbm
+	if (platform == EGL_PLATFORM_GBM_KHR) {
+		if (!check_egl_ext(client_exts_str, "EGL_KHR_platform_gbm")) {
```

Comment:
Can we check for `EGL_MESA_platform_gbm` as well?

https://www.khronos.org/registry/EGL/extensions/MESA/EGL_MESA_platform_gbm.txt

Reply:
Well, I guess this isn't a big deal, since only ancient Mesa version would have `EGL_MESA_platform_gbm` but not `EGL_KHR_platform_gbm`.

Reply:
On some devices that if it is not use MESA driver. In the case, Do we should be check the EGL_MESA_platform_gbm?

Reply:
I don't think non-Mesa drivers have the MESA ext